### ISSUE TITLE
Accessibility: "Advanced Settings" cannot be expanded via keyboard

### ISSUE THREAD

**Issue by krallowski** [humhub/wiki#433]
# Description
According to WCAG guidelines, interactive elements must be reached by keyboard. Maybe I am wrong, but it seems to me that I cannot reach the "Advances Settings" section via keyboard because focus the div-element to expand it with my keyboard.

# Steps to reproduce
1. create a new wiki page
2. Press tab until you should reach "Advanced settings"

# Expected behaviour
The Advanced settings blocks gets the focus at some poiunt and I can somehow expand the Advanced settings eg with pressing enter.

# Current behaviour
The block gets no focus.

# Possible solution
Replace the div with a button element or add another semantic interactive element maybe here: https://github.com/humhub/humhub/blob/899e244556194be9cbb825c74ff8e1a8feaaae6e/protected/humhub/widgets/form/ActiveForm.php#L68

**Comment by yurabakhtin** [comment]
@luke- PR https://github.com/humhub/humhub/pull/8260:

https://github.com/user-attachments/assets/e5c32471-ed54-4d9c-9dd1-28d7b4920ca9

https://github.com/user-attachments/assets/ef42f3bd-6225-4921-a375-c16fed668d59
